packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -590,27 +589,18 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
                 validate: (value) => value !== null,
               }}
               render={({ field: { value, onChange, ref } }) => (
-                <RadioButtonsContainer>
-                  <RadioButton
-                    ref={ref}
-                    value="false"
-                    label="All audiences"
-                    onChange={() => onChange(false)}
-                    selectedValue={value?.toString()}
-                    error={!!errors.isExplicit}
-                    helperText={errors.isExplicit ? 'Content rating must be selected' : ''}
-                    disabled={videoFieldsLocked}
-                  />
-                  <RadioButton
-                    value="true"
-                    label="Mature"
-                    onChange={() => onChange(true)}
-                    selectedValue={value?.toString()}
-                    error={!!errors.isExplicit}
-                    helperText={errors.isExplicit ? 'Content rating must be selected' : ''}
-                    disabled={videoFieldsLocked}
-                  />
-                </RadioButtonsContainer>
+                <RadioButtonGroup
+                  ref={ref}
+                  options={[
+                    { label: 'All audiences', value: 'false' },
+                    { label: 'Mature', value: 'true' },
+                  ]}
```

Comment:
I set my video as mature, but nothing happened when I published my changes, and the transaction was completed. Could you check what's wrong?

Reply:
ok, ignore that. It's not your fault. This should be fixed once we merge https://github.com/Joystream/atlas/pull/2854